### ISSUE TITLE
"Connecting..." forever

### ISSUE THREAD

**Issue by tparadox** [yakyak/yakyak#1069]
I just downloaded the latest version for Windows 10 for the first time, signed into my Google account, and Yakyak now just sits there endlessly trying to connect. All the related issues I can find here seem to be closed as stale with no helpful information.

**Comment by servusDei2018** [comment]
If you log into your Google account, is there any information there? Check your [security alerts](https://myaccount.google.com/intro/security).

**Comment by kohbo** [comment]
I'm having this same issue. I'm running on Windows 10 x64.

I tried direct running of the executable from the 1.5.3, 1.5.2, 1.5.1, and 1.5.4Beta releases. I also did a clean build of 1.5.3 from source and a checkout from master branch.

All of them would stay stuck at connecting after authorizing my Google login. Nothing is printed in the Inspection Tool console or the Windows command line that indicates an error has occurred. I am behind an enterprise firewall and have their http proxy configured. It looks like the proxy is correctly resolved by YakYak. There are no security events on my google account.
![image](https://user-images.githubusercontent.com/6541769/71211534-f595b000-2274-11ea-98eb-44d44a50cf98.png)

Here is the output when running YakYak.exe:
![image](https://user-images.githubusercontent.com/6541769/71211787-78b70600-2275-11ea-853e-d43c21fc1a87.png)
